README.md

```diff
@@ -110,41 +110,74 @@ This sample shows how to create custom AnalyzeRules for use within the Analyze w
+3. In the “Content Packing & Loading” schema:
+   - Set the “Build & Load Paths” to the default local paths (LocalBuildPath and LocalLoadPath). At runtime we will configure the load paths to use asset pack locations (see AddressablesInitSingleton.cs).
+   - **Note**: Since the Google Play Console doesn't provide remote URLs for uploaded content, it is not possible to use remote paths or the Content Update workflow for content assigned to asset packs. Remote content will need to be hosted on a different CDN.
+   - In Advanced Options > Asset Bundle Provider use the “Play Asset Delivery Provider”. This will download asset packs before loading bundles from them.
+
+##### Build Addressables
+Build Addressables using the custom “Play Asset Delivery” build script. In the Addressables Groups Window, do Build > New Build > Play Asset Delivery.
+This script will:
+1. Create the config files necessary for creating [custom asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#custom-asset-packs)
+   - Each asset pack will have a directory named “{asset pack name}.androidpack” in 'Assets/PlayAssetDelivery/Build/CustomAssetPackContent'.
+   - **Note**: All .bundle files created from a previous build will be deleted from 'Assets/PlayAssetDelivery/Build/CustomAssetPackContent'. 
+   - Each .androidpack directory also contains a ‘build.gradle’ file. If this file is missing, Unity will assume that the asset pack uses "on-demand" delivery.
+2. Generate files that store build and runtime data that are located in 'Assets/PlayAssetDelivery/Build':
+   - Create a 'BuildProcessorData.json' file to store the build paths and .androidpack paths for bundles that should be assigned to custom asset packs. At build time this will be used by the PlayAssetDeliveryBuildProcessor to relocate bundles to their corresponding .androidpack paths.
+   - Create a 'CustomAssetPacksData.json' file to store custom asset pack information to be used at runtime.
+
+##### Configure Runtime Scripts
+Prepare Addressables to load content from asset packs at runtime (see AddressablesInitSingleton.cs):
+1. Make sure that the generated asset packs are downloaded.
+2. Configure our custom InternalIdTransformFunc, which converts internal ids to their respective asset pack location.
+3. Load all custom asset pack data from the "CustomAssetPacksData.json".
+
+Once configured, you can load assets using the Addressables API (see LoadObject.cs).
+
+##### Build the Android App Bundle
+Assuming that you have configured the build settings mentioned here [Configure Build & Player Settings](#Configure-Build-&-Player-Settings), to build the Android App Bundle do File > Build Settings > Build. 
```

Comment:
Grammar change:
"When you have configured the build settings according to the [Configure Build & Player Settings](#Configure-Build-&-Player-Settings) instructions, go to **File** > **Build Settings** and select **Build** to build the Android App bundle."